parser_library/src/diagnostic_op.cpp

```diff
@@ -0,0 +1,2586 @@
+/*
+ * Copyright (c) 2019 Broadcom.
```

Comment:
```suggestion
 * Copyright (c) 2024 Broadcom.
```

Reply:
This one contains mostly stuff from diagnostic.cpp as well.